action.yml

```diff
@@ -1,11 +1,15 @@
 name: "Python Dependency Installation"
 description: "Install Python dependencies from requirements.txt file"
 inputs:
-  path: # id
```

Comment:
Instead of removing old input names, it's best to use a `deprecationMessage` (with this setting, GitHub hints end-users to update gracefully, whenever they run your action) like I did here: https://github.com/pypa/gh-action-pypi-publish/blob/79739dc/action.yml#L12-L23. Of course, this requires a bit of logic elsewhere, to use the old value as a fallback.